tests/unit/storage_v3.cpp

```diff
@@ -711,20 +711,20 @@ TEST_P(StorageV3, VertexDeleteLabel) {
 
     // Check whether label 5 exists
     ASSERT_FALSE(vertex->HasLabel(label5, View::OLD).GetValue());
-    ASSERT_EQ(vertex->HasLabel(label5, View::NEW).GetError(), Error::DELETED_OBJECT);
+    ASSERT_EQ(vertex->HasLabel(label5, View::NEW).GetError(), SHARD_ERROR(common::ErrorCode::DELETED_OBJECT));
```

Comment:
I think implementing an `operator==` for `ShardError` and `ErrorCode` would also make these tests more readable, because with the shard error we shouldn't depend on the source/message.